resources/abbreviations.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Idk where you are going with this little database lookup file of yours, but I will allow it if you call it what it is, a set of output filters.

I am working on changing the corrections into an input filter mapping. It is done, just building out the list. This is effectively that for output, except you have multiple different groupings.